account_invoice_report_service/views/report_invoice_delivery.xml

```diff
@@ -17,74 +17,81 @@
     </template>
 
     <template id="external_layout_invoice_delivery">
-        <t t-if="not o and doc">
-            <t t-set="o" t-value="doc" />
-        </t>
+        <t t-call="web.html_container">
+            <t t-if="not o and doc">
+                <t t-set="o" t-value="doc" />
+            </t>
 
-        <t t-if="o and 'company_id' in o">
-            <t t-set="company" t-value="o.company_id.sudo()" />
-        </t>
-        <t t-if="not o or not 'company_id' in o">
-            <t t-set="company" t-value="res_company" />
-        </t>
+            <t t-if="o and 'company_id' in o">
+                <t t-set="company" t-value="o.company_id.sudo()" />
+            </t>
+            <t t-if="not o or not 'company_id' in o">
+                <t t-set="company" t-value="res_company" />
+            </t>
```

Comment:
@ChrisOForgeFlow The non 4-spaces indentation makes this differ, can you please keep always the 4 spaces in order to only see the real changes.

Reply:
This is changed by pre-commit script, if you see bellow on script I added tag <t t-call="web.html_container"> for call correctly assets for reports, because this change indentation of children

Reply:
You're right, so please, squash the commits and I'll merge it. Thanks!